src/status_im/signals/core.cljs

```diff
@@ -50,6 +51,20 @@
     {:db (assoc db :peer-stats peer-stats
                 :peers-count (count (:peers peer-stats)))}))
 
+(defn hide-sheet-and-dispatch [event]
+  (re-frame/dispatch [:bottom-sheet/hide])
+  (re-frame/dispatch event))
+
+(defn handle-local-pairing-signals [signal-type]
+  (log/info "localpairing signal received ====> " signal-type)
```

Comment:
`localpairing` -> `local pairing`?

Reply:
its just a log message :)

Reply:
yeah but it's good to keep these things neat and tidy.